x-pack/plugins/alerting/server/constants/translations.ts

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { i18n } from '@kbn/i18n';
+
+export const translations = {
+  taskRunner: {
+    warning: {
+      maxExecutableActions: i18n.translate(
+        'xpack.alerting.taskRunner.warning.maxExecutableActions',
+        {
+          defaultMessage:
+            'Rule execution has reached the maximum number of executable actions limit',
```

Comment:
As a novice user of alerting features, I have these questions:

- Do I need to care *which* rule execution hit this limit?
- Does it affect subsequent executions?
- Do I need to know which actions were not triggered?

I think it would be helpful if we can address some of those questions in this message (and also avoid the term "execution" if possible, per above). For example:

> The maximum number of actions was reached [when? which alerts were affected?]; excess actions [which?] were not triggered.

Reply:
Thanks for the explanation!  I think that suggested sentence is good!